docs/source/machine_translation.rst

```diff
@@ -0,0 +1,255 @@
+
+Machine Translation Task Tutorial
+=================================
+
+This tutorial will guide you through designing Machnie Translation Task
+experiments in Neural Monkey. It assumes that you already read the
+Post-Editing tutorial.
+
+The goal of this task is to translate sentences from one language into
+another. For this tutorial we will use data from IT translation task of WMT
+16 on English to Czech direction.
+
+
+Part I. - The Data
+--------------------
+
+We are going to use the data for WMT 16 shared IT Machine Translation task. You
+can get them at the `WMT 16 IT Translation Task
+<http://www.statmt.org/wmt16/it-translation-task.html>`_ and there download
+Batch1 and Batch2 answers and Batch3 as a testing set. Or directly `here
+<http://ufallab.ms.mff.cuni.cz/~popel/batch1and2.zip>`_ and
+`testset <http://ufallab.ms.mff.cuni.cz/~popel/batch3.zip>`_.
+
+Note: Since we are using only small data for example which are not enough
+ for real-life machine translation training.
+
+When we open the archive we find several files for different languages, we
```

Comment:
Předtím se psalo v druhé osobě, najednou je to v množném číslo, chtělo by to sjednotit.